scripts/setup_smtp.py

```diff
@@ -20,18 +24,23 @@ def main(env, host, user, password):
     values = {
         "name": "Default SMTP",
         "smtp_authentication": "login",
-        "smtp_encryption": "starttls",
-        "smtp_port": 587,
+        "smtp_encryption": encryption,
+        "smtp_port": port,  # 587
         "sequence": 9999999,
         "smtp_host": host,
         "smtp_user": user,
         "smtp_pass": password,
         "private": True,
+        "from_filter": user.split("@")[1],
```

Comment:
Do we need to allow for "stupid" users, who do not enter a proper email address?

Reply:
In this case the user would be a devops person, I could add a regex to check if it's a correctly formatted address but I think it's a slim case as this is automatically generated (for Curq) anyways.